engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -319,19 +304,16 @@ private void renderShadowMap() {
     public void renderWorldReflection() {
         PerformanceMonitor.startActivity("Render World (Reflection)");
 
-        DefaultRenderingProcess.getInstance().beginRenderReflectedScene();
-        GL11.glCullFace(GL11.GL_FRONT);
+        LwjglRenderingProcess.getInstance().beginRenderReflectedScene();
         playerCamera.setReflected(true);
 
-        playerCamera.lookThroughNormalized();
+        playerCamera.lookThroughNormalized(); // we don't want the reflected scene to be bobbing or moving with the player
         backdropRenderer.render(playerCamera);
         playerCamera.lookThrough();
 
         chunkShader.activateFeature(ShaderProgramFeature.FEATURE_USE_FORWARD_LIGHTING);
 
         if (renderingConfig.isReflectiveWater()) {
-            GL11.glEnable(GL_LIGHT0);
```

Comment:
Random thoughts that probably do not relate but this reminded me of it: There are some known water lighting issues, in some cases models turn black (or white) like after applying updated graphic settings while in-game, and sunlight can reflect funny on water even if the sun is behind a hill.

I don't imagine this change will affect any of those - but mentioning anyway :-) Looks fine to me!